<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mini Project_00 Ver1.0</string>
  </property>
  <widget class="QLabel" name="lable_LED">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>300</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>D2Coding</family>
     <pointsize>32</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style: solid;
border-width: 2px;
border-color: yellow;
border-radius: 3px</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>LED Controller</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lable_Ultra">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>70</y>
     <width>500</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>D2Coding</family>
     <pointsize>32</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style: solid;
border-width: 2px;
border-color: yellow;
border-radius: 3px</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>Ultrasonic Controller</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lable_Buzz">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>70</y>
     <width>400</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>D2Coding</family>
     <pointsize>32</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style: solid;
border-width: 2px;
border-color: yellow;
border-radius: 3px</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>Buzzer Controller</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btnRedOnly">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>300</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>D2Coding</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>RED ONLY</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBlue">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>300</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>D2Coding</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>BLUE ON</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnGreen">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>300</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>D2Coding</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>GREEN ON</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnRed">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>300</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>D2Coding</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>RED ON</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnLEDoff">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>710</y>
     <width>300</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>D2Coding</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ALL LED OFF</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnGreenOnly">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>470</y>
     <width>300</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>D2Coding</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>GREEN ONLY</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBlueOnly">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>550</y>
     <width>300</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>D2Coding</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>BLUE ONLY</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnLEDon">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>630</y>
     <width>300</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>D2Coding</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ALL LED ON</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnUltraOff">
   <property name="geometry">
    <rect>
     <x>425</x>
     <y>230</y>
     <width>350</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>D2Coding</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ULTRASONIC OFF</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnUltraOn">
   <property name="geometry">
    <rect>
     <x>425</x>
     <y>150</y>
     <width>350</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>D2Coding</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ULTRASONIC ON</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBuzzSchool">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>230</y>
     <width>300</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>D2Coding</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>School Bell</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBuzzOn">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>150</y>
     <width>300</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>D2Coding</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>BUZZER ON</string>
   </property>
  </widget>
  <widget class="QLabel" name="picPEPE">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>320</y>
     <width>561</width>
     <height>470</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_Display">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>1180</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>D2Coding</family>
     <pointsize>32</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>Hello World</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBuzzOff">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>310</y>
     <width>300</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>D2Coding</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>BUZZER OFF</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
